### PR TITLE
Feature/369/recipe directions

### PR DESCRIPTION
**Pull request by nickharriscodes**
### Describe The Problem Being Solved:
Allow user to enter custom step by step instructions/directions to their recipes. 

### Checklist For Submitter

* [] I have documented all new functionality
* [x] I have screenshotted new UI changes and attached them to this PR

Screen Shot 2019-10-30 at 8 15 54 PM

### CODE REVIEW

File: frontend/src/components/EditRecipe/EditRecipeForm.js [hunk 1]
```diff
@@ -76,6 +98,14 @@ const EditRecipeForm = ({ history }) => {
           </select>
         </div>
       </div>
+      <div>
+        <EditDirections
+          directions={directions}
+          handleDirectionsAdd={handleDirectionsAdd}
+          handleDirectionsChange={handleDirectionsChange}
+          handleDirectionsRemove={handleDirectionsRemove}
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.js]: Same question as the ingredients; what do you think about passing in `setDirections` here instead and moving the `handle*` methods into the directions component?

File: frontend/src/containers/RecipePage/Recipe.js [hunk 2]
```diff
@@ -68,7 +68,6 @@ class Recipe extends Component {
         </button>
       </div>
     );
-
```
Comment (vhoof) [inline frontend/src/containers/RecipePage/Recipe.js]: I think it's worth hanging on to this empty line between methods.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (vhoof) [inline package-lock.json]: I'd remove this file.

File: frontend/src/components/EditRecipe/EditDirections/EditDirections.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      <div className="directions-btn-container">
+        <button className="btn navbar-btn btn-lg add-directions-button" type="button" onClick={() => handleDirectionsAdd()}>
+          + ADD
+        </button>
+      </div>
+    </div>
+  </div>
+);
+
+
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditDirections/EditDirections.js]: There's an extra empty line here.

File: frontend/src/components/EditRecipe/EditRecipeForm.js (hunk 1 above)
Comment (nickharriscodes) [reply frontend/src/components/EditRecipe/EditRecipeForm.js]: @vhoof I was having trouble/getting an error when I tried that approach. setDirections was not recognized as a function when called in the child (EditDirections) component. I'll try again today and see if I can figure it out.

Comment (nickharriscodes) [reply frontend/src/components/EditRecipe/EditRecipeForm.js]: @vhoof Not sure what the error was that I was running into yesterday. The latest commit should address this & work properly.

**Pull request merged**